extensions/logging-json/deployment/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-logging-json-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+
+    <artifactId>quarkus-logging-json-deployment</artifactId>
+    <name>Quarkus - JSON Logging - Deployment</name>
```

Comment:
It's minor but maybe: `Quarkus - Logging - JSON - Deployment` in case we have several logging components in the future?

Same for the parent and runtime poms obviously.

Reply:
Sure.